tuner/examples/simple/simple_tuner.py

```diff
@@ -114,15 +122,20 @@ def main():
             return
 
         print("Benchmarking compiled dispatch candidates...")
+        logging.info(f"Summarization about top dispatch candidates:")
```

Comment:
No need for fstring, we don't print any variables here. Also, the grammar is weird. The summary is not ready at this point, so how about we just print the same thing as the print above?